app/templates/home.html

```diff
@@ -9,5 +9,7 @@ <h1>Welcome To Tobu Pengin</h1>
 {% endblock %}
 
 {% block rightblock %}
-    {% include "right_window/important_info.html" %}
+    <div class="important-info-card">
+        {% include "right_window/important_info.html" %}
+    </div>
```

Comment:
I'm sure this could be handled several different ways. Just wanted a quick/easy way to demonstrate handling the styling down the tree.